compiler/rustc_windows_rc/src/lib.rs

```diff
@@ -0,0 +1,145 @@
+//! A build script dependency to create a Windows resource file for the compiler
+//!
+//! Uses values from the `CFG_VERSION` and `CFG_RELEASE` environment variables
+//! to set the product and file version information in the Windows resource file.
+use std::{env, ffi, fs, path, process};
+
+use cc::windows_registry;
+
+/// The template for the Windows resource file.
+const RESOURCE_TEMPLATE: &str = include_str!("../rustc.rc.in");
+
+const VFT_APP: u32 = 0x00000001; // VFT_APP
+const VFT_DLL: u32 = 0x00000002; // VFT_DLL
+
+#[derive(Debug, Clone, Copy)]
+#[repr(u32)]
+pub enum VersionInfoFileType {
+    App = VFT_APP,
+    Dll = VFT_DLL,
+}
```

Comment:
A quick scan seems to indicate that these constant are only used in this enum. If so, then perhaps they can be inlined?